debian/patches/PluginSecurityManager.cs.patch

```diff
@@ -0,0 +1,101 @@
+--- a/Emby.Server.Implementations/Security/PluginSecurityManager.cs
```

Comment:
Let's avoid duplicating code and keep the patches/replacements in a single location in the repo. 

Use a build script to copy them if needed.

Reply:
Will need to change the format; this is a unified diff in Debian format, versus the main patch being the other format (manually converted). Might be worthwhile to replace the main patch with this one instead?